WordCounter/WordCounter.cs

```diff
@@ -8,22 +8,63 @@ namespace WordCounter
 {
     class WordCounter
     {
-        public WordCounter(string inputString)
+        private Dictionary<string, int> dict;
```

Comment:
There's no need for this method to hold on to this dictionary.  It can just be returned by the method that does the work.